typescript/packages/subsurface-viewer/src/layers/axes2d/axes2DLayer.ts

```diff
@@ -460,15 +489,35 @@ export default class Axes2DLayer extends Layer<Axes2DLayerProps> {
 
         const { viewMatrix } = this.context.viewport;
 
-        const mh = this.props.marginH * pixel2world;
-        const mv = this.props.marginV * pixel2world;
-
         const viewport_bounds_w = this.context.viewport.getBounds(); //bounds in world coordinates.
         const xBoundsMin = viewport_bounds_w[0];
         const xBoundsMax = viewport_bounds_w[2];
         const yBoundsMin = viewport_bounds_w[1];
         const yBoundsMax = viewport_bounds_w[3];
 
+        const [useExponential, useFracDigits, maxLength] = UseExponential(
```

Comment:
Are you deciding whether to use scientific format or not depending on the available space?

Reply:
No. Its decided by the size of the numbers. Huge numbers will be in scientific. But they have to be quite big as sceitific also takes up quite a few characetrs (ex: -1.23e12 = 8 chearacters.)